decidim-surveys/README.md

```diff
@@ -0,0 +1,24 @@
+# Decidim::Surveys
+
+The Surveys module adds one of the main features of Decidim: allows users to contribute to a particiaptory process by creating surveys.
```

Comment:
s/particiaptory/participatory

And the description is wrong, admins can create surveys ad users can participate in them.